.github/workflows/gluafixer.yml

```diff
@@ -0,0 +1,63 @@
+name: Lint GLua
+
+on:
+  workflow_call:
+    inputs:
+      config:
+        type: string
+        required: false
+        default: ""
+        description: "A URL or path (relative to your project root) of the glualint config"
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: project
+
+      - name: Prepare GLuaFixer
+        run: |
+          cd $GITHUB_WORKSPACE
+          curl -s https://api.github.com/repos/FPtje/GLuaFixer/releases/latest \
+          | grep "browser_download_url.*linux" \
+          | cut -d : -f 2,3 \
+          | tr -d \" \
```

Comment:
This can be quite sensitive to changes to the Github interface. Is this the best way to go, or is there also some interface that can return e.g. JSON?

Reply:
Fair question. this URL does return json, it's just doing a stupid parsing instead of a proper parsing.

I changed it to use `jq`, so we're actually parsing the JSON instead of searching for text 👍